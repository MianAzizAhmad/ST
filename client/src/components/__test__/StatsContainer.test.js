import React from 'react';
import StatsContainer from './../StatsContainer';
import "@testing-library/jest-dom/extend-expect"
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

const mockStore = configureStore([]);

describe('StatsContainer Test Case { ', ()=>{

    let store;
    let component;
    
    beforeEach(() => {
        store = mockStore({
            pending: '1' , declined: '1', interview: '1',
        });
        component = renderer.create(<Provider store={store}><StatsContainer ></StatsContainer></Provider>).toJSON();
    });

    

    it(' Snap 1 }', ()=>{
        expect(component).toMatchSnapshot();
    });
});