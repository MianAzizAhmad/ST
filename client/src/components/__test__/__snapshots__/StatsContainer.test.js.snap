// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsContainer Test Case {   Snap 1 } 1`] = `undefined`;
